src/main/java/command/ViewCommand.java

```diff
@@ -18,18 +18,35 @@ public ViewCommand(String args) {
         suspects = args.split(" ");
     }
 
+    /**
+     * Executes the view command. If there are suspect(s) specified by the user,
+     * prints already searched clues relating to them. Else print all clues that
+     * have been searched.
+     *
+     * @param ui            used to communicate with the user.
+     * @param investigation contains method to query the already searched clues.
+     * @param sceneList     not used in this method but passed in as a standard of
+     *                      command execution.
+     */
     @Override
     public void execute(Ui ui, Investigation investigation, SceneList sceneList) {
         ui.printViewingCheckedCluesMessage();
+        findSearchedClues(ui, investigation);
+    }
+
+    /**
+     * Finds the already searched clues for suspects specified by the user.
+     * 
+     * @param ui used to communicate with the user.
+     * @param investigation contains method to query the already searched clues.
+     */
+    private void findSearchedClues(Ui ui, Investigation investigation) {
```

Comment:
I think can standardize if want to put spacing to align the description. I think the params description need start with capital letter.